incubator/jaeger/templates/agent-ds.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
```

Comment:
I guess I confused you. Maybe I was not clear enough. Sorry. I did not want you to change the names, only the component labels. Do it like so throughout the chart:

`name: "{{ template "jaeger.fullname" . }}-agent"`

`component: agent`

Reply:
Ahhh I see. Will provide an update shortly.

Reply:
This has been addressed.